apps/sms/views/shared/js/navigation.js

```diff
@@ -106,7 +105,7 @@ var Navigation = {
     var currentArgs = currentPanel.args || {};
 
     for (var arg in args) {
-      if (args[arg] !== currentArgs[arg]) {
+      if (args[arg] != currentArgs[arg]) {
```

Comment:
can't we just use force threadId to be a number after parsing it as string? threadId = +threadId ?